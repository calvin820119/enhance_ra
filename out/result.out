0.310000 0.000000 0.000000 -1.#IND00 -1.#IND00 -1.#IND00 0
0.310000 0.000000 0.000000 -1.#IND00 -1.#IND00 -1.#IND00 0
0.310000 0.000000 0.000000 -1.#IND00 -1.#IND00 -1.#IND00 0
0.310000 0.000000 0.000000 -1.#IND00 -1.#IND00 -1.#IND00 0
0.310000 0.000000 0.000000 -1.#IND00 -1.#IND00 -1.#IND00 0
0.140000 0.000000 0.000000 -1.#IND00 -1.#IND00 -1.#IND00 0
9.950000 0.000000 0.000000 -1.#IND00 -1.#IND00 -1.#IND00 0
11.550000 0.000000 0.000000 -1.#IND00 -1.#IND00 -1.#IND00 0
9.890000 0.000000 0.000000 -1.#IND00 -1.#IND00 -1.#IND00 0
9.743000 0.000000 0.000000 -1.#IND00 -1.#IND00 -1.#IND00 0
9.743000 0.000000 0.000000 -1.#IND00 -1.#IND00 -1.#IND00 0
9.743000 0.000000 0.000000 -1.#IND00 -1.#IND00 -1.#IND00 0
