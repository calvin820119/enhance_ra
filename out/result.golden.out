1.000000
1.000000
0.999983
0.999925
0.999890
0.999517
0.998971
0.997305
0.994706
0.991050
